Cyral/README.md

```diff
@@ -49,7 +49,30 @@ This is an example of a simple query that returns average execution time.
 
 ### Install
 
-After configuring collection you can [install](https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library) the SumoLogic App from Cyral from App Catalog and use the same source category configured while creating the source.
+After configuring the collection you can [install](https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library) the SumoLogic App from Cyral from App Catalog and use the same source category configured while creating the source.
+
+#### Post Install Configuration
+
+Some panels in the dashboard make use of a lookup file called `CategoryLookupTable.csv` located in the resources directory of this repo that must also be installed.
+
+1. Login to your Sumo Logic instance as an Admin
+2. Switch to the `Library View`
+3. Within the `Library View`, set the View As to `Content Administrator`
+4. Within the `Admin Recommended` Folder, click on the `Add New` button
+5. From the `Add New` button, select `New Lookup`
+6. In the resulting Lookup Configuration screen, use the following settings
+  - Name: CategoryLookup
+  - Set a TTL: No
+  - Choose a size limit handling option: Delete Old Data
+  - Create Lookup Table: Upload File
```

Comment:
Also specify which fields needs to be primary fields one can only lookup by primary fields present in csv.